package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@taiga-ui/components",
-    "version": "3.58.0",
+    "version": "4.0.0",
```

Comment:
```
npm run release:major
```

will do the following:
<img width="1099" alt="Screenshot 2023-12-12 at 18 09 50" src="https://github.com/taiga-family/taiga-ui/assets/35179038/5b54fec0-a66e-4d2d-bef6-e28f669f850d">

Reply:
for now this is necessary to create snapshots for e2e

Reply:
although no other PRs will be sent to this PR, reverted to 3.58